bin/postgres-ha/bootstrap-postgres-ha.sh

```diff
@@ -61,14 +61,22 @@ initialization_monitor() {
 
         if [[ "${PGHA_INIT}" == "true" ]]
         then
+
+            if [[ "${PGHA_STANDBY}" != "true" ]]
+            then
+                primary_endpoint="master"
+            else
+                primary_endpoint="standby_leader"
+            fi
```

Comment:
I'd recommend going simpler:

```suggestion
            primary_endpoint="master"
            if [[ "${PGHA_STANDBY}" == "true" ]]
            then
                primary_endpoint="standby_leader"
            fi
```